files/jquery.smoothstate/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "smoothstate",
+  "repo": "miguel-perez/smoothState.js",
+  "files": {
+    "include": ["jquery.smoothState.min.js", "src/jquery.smoothState.js"]
```

Comment:
Hm. I do check at my NodeJS app whether the app is in a production environment, and if yes I just append `.min` to my CSS and JS files. This is so I can debug easier when developing. I also saw that lots of libraries on JSDelivr also provide the unminified files, so I thought its normal. But I'll remove it.